model/src/main/kotlin/config/Resolutions.kt

```diff
@@ -29,10 +29,18 @@ class Resolutions(
      * Resolutions for issues with the analysis or scan of the projects in this repository and their dependencies.
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val errors: List<ErrorResolution> = emptyList()
+    val errors: List<ErrorResolution> = emptyList(),
+    /**
+     * Resolutions for license policy violations.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val evaluatorErrors: List<EvaluatorErrorResolution> = emptyList()
```

Comment:
Please also update Configuration.md according to this change.

Reply:
Do we need this as part of this PR regarding urgency?

Reply:
Ok.